cartography/client/core/tx.py

```diff
@@ -0,0 +1,50 @@
+from typing import Any
```

Comment:
@marco-lancini, @ryan-lane, @mpurusottamc - hey, so I've created this set of functions so that other Python clients might have an easier time pulling data from a cartography-based graph (since these functions handle the boilerplate transaction logic)

I'd like to write maybe 1-3 sample Python functions using these to show users what they can do. For example maybe I'd include something like `get_ec2_instances_by_account()`. Any ideas on what would be good examples? Anything from https://github.com/marco-lancini/cartography-queries/ or any other places that you've found particularly compelling when showing your projects to people? :)